west.yml

```diff
@@ -47,7 +47,7 @@ manifest:
       revision: 6835bfc741bf15e98fb7971293913f770df6081f
       path: modules/hal/esp-idf
     - name: fatfs
-      revision: 9ee6b9b9511151d0d64a74d532d39c6f2bbd4f16
+      revision: 79a949a19fa3e016198542b1f8845355670bfb21
```

Comment:
Using the pull request for revision will help the CI: https://docs.zephyrproject.org/latest/guides/modules.html#changes-to-existing-modules